FND/Accounting/AccountingEquity.rdf

```diff
@@ -159,6 +159,14 @@
 		<skos:definition>assets received from investors in exchange for an ownership interest</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fnd-acc-aeq;PhysicalAsset">
```

Comment:
Should have synonym of RealAsset - in fact that should be the preferred term from what I've read.

Reply:
There is apparently a distinction - real asset is often used interchangeably with finished goods, whereas physical asset includes commodities (which is where we need this term).

Reply:
Real asset also includes commodities. This is just one of many definitions that support this https://www.investopedia.com/terms/r/realasset.asp#:~:text=Key%20Takeaways,all%20types%20of%20real%20assets.